modules/core/client/views/footer.client.view.html

```diff
@@ -0,0 +1,10 @@
+<div class="container" ng-controller="FooterController as vm">
+  <div class="row">
+    <div class="col-md-12">
+      <hr/>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-md-4 col-md-offset-4">&copy; {{vm.year}} MEAN.js | All rights reserved</div>
```

Comment:
should use [app.title](https://github.com/meanjs/mean/blob/master/config/env/default.js#L5) instead of the MEAN.js text.